// @acala-network/types/interfaces/definitions.js
"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "primitives", {
  enumerable: true,
  get: function () {
    return _definitions.default;
  }
});
Object.defineProperty(exports, "accounts", {
  enumerable: true,
  get: function () {
    return _definitions2.default;
  }
});
Object.defineProperty(exports, "auctionManager", {
  enumerable: true,
  get: function () {
    return _definitions3.default;
  }
});
Object.defineProperty(exports, "cdpEngine", {
  enumerable: true,
  get: function () {
    return _definitions4.default;
  }
});
Object.defineProperty(exports, "collatorSelection", {
  enumerable: true,
  get: function () {
    return _definitions5.default;
  }
});
Object.defineProperty(exports, "dex", {
  enumerable: true,
  get: function () {
    return _definitions6.default;
  }
});
Object.defineProperty(exports, "evm", {
  enumerable: true,
  get: function () {
    return _definitions7.default;
  }
});
Object.defineProperty(exports, "homa", {
  enumerable: true,
  get: function () {
    return _definitions8.default;
  }
});
Object.defineProperty(exports, "homaValidatorList", {
  enumerable: true,
  get: function () {
    return _definitions9.default;
  }
});
Object.defineProperty(exports, "incentives", {
  enumerable: true,
  get: function () {
    return _definitions10.default;
  }
});
Object.defineProperty(exports, "loans", {
  enumerable: true,
  get: function () {
    return _definitions11.default;
  }
});
Object.defineProperty(exports, "nft", {
  enumerable: true,
  get: function () {
    return _definitions12.default;
  }
});
Object.defineProperty(exports, "nomineesElection", {
  enumerable: true,
  get: function () {
    return _definitions13.default;
  }
});
Object.defineProperty(exports, "runtime", {
  enumerable: true,
  get: function () {
    return _definitions14.default;
  }
});
Object.defineProperty(exports, "stakingPool", {
  enumerable: true,
  get: function () {
    return _definitions15.default;
  }
});
Object.defineProperty(exports, "renvmBridge", {
  enumerable: true,
  get: function () {
    return _definitions16.default;
  }
});
Object.defineProperty(exports, "support", {
  enumerable: true,
  get: function () {
    return _definitions17.default;
  }
});

var _definitions = _interopRequireDefault(require("./primitives/definitions"));

var _definitions2 = _interopRequireDefault(require("./accounts/definitions"));

var _definitions3 = _interopRequireDefault(require("./auctionManager/definitions"));

var _definitions4 = _interopRequireDefault(require("./cdpEngine/definitions"));

var _definitions5 = _interopRequireDefault(require("./collatorSelection/definitions"));

var _definitions6 = _interopRequireDefault(require("./dex/definitions"));

var _definitions7 = _interopRequireDefault(require("./evm/definitions"));

var _definitions8 = _interopRequireDefault(require("./homa/definitions"));

var _definitions9 = _interopRequireDefault(require("./homaValidatorList/definitions"));

var _definitions10 = _interopRequireDefault(require("./incentives/definitions"));

var _definitions11 = _interopRequireDefault(require("./loans/definitions"));

var _definitions12 = _interopRequireDefault(require("./nft/definitions"));

var _definitions13 = _interopRequireDefault(require("./nomineesElection/definitions"));

var _definitions14 = _interopRequireDefault(require("./runtime/definitions"));

var _definitions15 = _interopRequireDefault(require("./stakingPool/definitions"));

var _definitions16 = _interopRequireDefault(require("./renvmBridge/definitions"));

var _definitions17 = _interopRequireDefault(require("./support/definitions"));
